_data/mentors-directory.yaml

```diff
@@ -18,6 +18,13 @@
       slack: https://opensourceday.slack.com/team/U026H8FRASJ
       ask-me: Open Source
       image: srishti-guleria.jpg
+    - name: Yogita Bhatia
+      github: https://github.com/Yogita98
+      linkedin: https://www.linkedin.com/in/yogita-bhatia/
+      twitter: https://twitter.com/YogitaBhatia20
+      slack: https://opensourceday.slack.com/team/U027YJPRDEW
+      ask-me: Git, Open Source, Communities, JavaScript, ReactJS, NodeJS, Bash, Python, DevOps 
+      image: yogita-bhatia.jpg
```

Comment:
Images needs to be the same hight and width to be displayed correctly. Could you please update the image uploaded?